fetch-pipeline-yamls.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Try to get to the root of the problem before doing workarounds. At least this script should be generic enough to fetch any kind of file.

Opened an issue https://github.com/mondeja/mkdocs-include-markdown-plugin/issues/198